src/Extension/Exception/DependentExtensionsExceptionHandler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extension\Exception;
+
+use Flarum\Foundation\ErrorHandling\HandledError;
+
+class DependentExtensionsExceptionHandler
+{
+    public function handle(DependentExtensionsException $e): HandledError
```

Comment:
Do we have a habit of shortening `$exception` to `$e` ? I personally don't like that trend very much.

Reply:
That seems to generally be the trend, although we have a few `$error`s as well

Reply:
Ok that's fine too